experimenter/experimenter/nimbus-ui/src/components/Summary/index.tsx

```diff
@@ -72,7 +72,9 @@ const Summary = ({ experiment, refetch }: SummaryProps) => {
       changelogMessage: CHANGELOG_MESSAGES.CANCEL_REVIEW,
       statusNext:
         experiment.status === NimbusExperimentStatusEnum.LIVE
-          ? NimbusExperimentStatusEnum.LIVE
+          ? experiment.isRollout
+            ? null
+            : NimbusExperimentStatusEnum.LIVE
           : null,
```

Comment:
Shouldn't `statusNext` always go to `null` when canceling a review, regardless of whether it's a rollout or an experiment?

<img width="628" alt="image" src="https://github.com/mozilla/experimenter/assets/119884/57a7d398-1f81-4b31-ab7d-6a50f5905135">
<img width="694" alt="image" src="https://github.com/mozilla/experimenter/assets/119884/d9c87c41-6193-45de-8f64-53899d3003f4">

Reply:
okay actually yes if its rollout/experiment is going from draft, we launch and we cancel it then it should go back to statusNext = null, but lets say if rollout/experiment is live, then it should go to statusNext as live, but setting it to live for rollout, doesn't let us make changes 🤔